lib/reporters/list.js

```diff
@@ -8,6 +8,12 @@
 
 var Base = require('./base');
 var inherits = require('../utils').inherits;
+var constants = require('../runner').constants;
```

Comment:
Since these are being for the event names...
```js
var events = require('../runner').constants;
```

Reply:
if/when this becomes ES6 code, it can be easily search/replace using a regex as `const {constants} = require('../runner')`.